pkg/opni/commands/rbac.go

```diff
@@ -169,6 +209,36 @@ func BuildRoleBindingsCreateCmd() *cobra.Command {
 	}
 }
 
+func BuildRoleBindingsUpdateCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "update <rolebinding-id> <role-id> <user-id>...",
+		Short: "Update a role binding",
+		Args:  cobra.MinimumNArgs(3),
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			if len(args) == 1 {
+				return completeRoles(cmd, args, toComplete)
```

Comment:
This doesn't look quite right, can you update the completion logic for this command as well as the update roles command?

Reply:
Should https://github.com/rancher/opni/blob/main/pkg/opni/commands/rbac.go#L149 also be `completeRoleBindings` instead of `completeRoles`?

Reply:
It needs both, depending on the current number of arguments. Based on the usage 
`Use:   "create <rolebinding-id> <role-id> <user-id>` 
it should complete rolebindings for arg 0, roles for arg 1, and arg 2+ should just be ShellCompDirectiveNoFileComp